bindings/ruby/opendal.gemspec

```diff
@@ -17,11 +17,20 @@
 
 # frozen_string_literal: true
 
-require_relative "lib/opendal_ruby/version"
+require "json"
 
 Gem::Specification.new do |spec|
   spec.name = "opendal"
-  spec.version = OpenDAL::VERSION
+  # RubyGems integrates and expects `cargo`.
```

Comment:
It's a bit surprising to use `json` and need to call `cargo` for this. This might make things a bit complicated during the release process. However, it's fine for now; we can address it later.

Reply:
I agree with you.

If my memory checks out, RubyGems can also read a version file. At least I can do this manually. But I didn't find Cargo.toml supporting this.